### PR TITLE
Story/ccls 1708 app case search

### PR DESCRIPTION
**Pull request by porritta**

### CODE REVIEW

File: src/main/resources/application-local.yml
Comment (PhilDigitalJustice) [inline src/main/resources/application-local.yml:1]: need to revert this file

**Review by PhilDigitalJustice — CHANGES_REQUESTED** [review]
A few comments, but the rest looks good

File: src/main/java/uk/gov/laa/ccms/caab/controller/application/search/CopyCaseSearchController.java [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+  private final ProviderService providerService;
+
+  private final ApplicationService applicationService;
+
+  private final ApplicationMapper applicationMapper;
+
+  private final CaseSearchCriteriaValidator searchCriteriaValidator;
+
+  protected static final String CURRENT_URL = "currentUrl";
```
[the remark this answers is not in the document]
Comment (porritta) [reply src/main/java/uk/gov/laa/ccms/caab/controller/application/search/CopyCaseSearchController.java:60]: ApplicationSearchController updated

File: src/main/resources/application-local.yml
Comment (porritta) [reply src/main/resources/application-local.yml:1]: done

**Review by PhilDigitalJustice — APPROVED** [review]

**Comment by PhilDigitalJustice** [comment]
LGTM

**Pull request merged**